Dockerfile

```diff
@@ -1,29 +1,39 @@
-FROM ubuntu:bionic
+FROM php:7.3-apache
 
-# TODO: think through optimizations here once this is working.
-RUN    apt-get update \
-    && apt --yes install software-properties-common \
-    && apt-add-repository --yes --update ppa:ansible/ansible \
-    && apt --yes install git ansible
+# Extensions we know LF requires/desires:
+# curl - already provided in image
+# mbstring - already provided in image
+# xdebug
+# intl
+# gd
+# mongodb
+#
+# see https://github.com/mlocati/docker-php-extension-installer
+COPY --from=mlocati/php-extension-installer /usr/bin/install-php-extensions /usr/local/bin/
```

Comment:
slick!  Is this simply a different way of addressing extensions?  Here's what I did for mongo...

```yaml
# install mongo extension and enable it
RUN pecl install mongodb && \
    docker-php-ext-enable mongodb
```

obviously there's more than one way to skin a cat and these may both work so it may be moot, I'm asking for my own knowledge.